test/python/fx_importer/sparse_test.py

```diff
@@ -158,7 +158,14 @@ def sparse_jit(f, *args, **kwargs):
     for a in args:
         if a.layout is torch.sparse_coo:
             xargs.append(a.values().numpy())
-            xargs.append(a.indices().numpy())
+            # Construct the additional position array required by MLIR with data
+            # array([0, nnz]).
+            nnz = a.indices().shape[1]
+            xargs.append(torch.tensor([0, nnz], dtype=a.indices().dtype).numpy())
+            # Transform a tensor<ndim x nnz> into [tensor<nnz> x ndim] to conform
+            # MLIR SoA COO representation.
+            for idx in a.indices():
+                xargs.append(idx.numpy())
```

Comment:
can this be done with a view change and then the append? also, does this generalize to n-dim COO?

Reply:
I am hoping that it is already a view change (but I am not sure).

I think it generalize to n-dim COO. Per torch.sparse documentation " the indices of specified elements are collected in indices tensor of size (ndim, nse)"

Reply:
I dumped the `data_ptr()` of `a.indices()` and `idx` used in the loop and confirm that they are alias.